moveit_setup_assistant/templates/moveit_config_pkg_template/launch/demo_gazebo.launch

```diff
@@ -0,0 +1,69 @@
+<launch>
```

Comment:
There is another option of editing the current `demo.launch` and add options to it to be able to launch `gazebo` too.

Reply:
I think this second option will make demo.launch easier to maintain, rather than having a fork for gazebo

Reply:
May I suggest to *not* change `demo.launch`? MoveIt packages currently already have quite a few dependencies that make them rather involved to deploy on headless machine (fi). If we start extending `demo.launch` to include Gazebo functionality, that would make Gazebo an `exec_depend` of every MoveIt package.

Gazebo definitely has its usages, but making it a hard dependency of a MoveIt package as it gets generated by the authoratitive tool for such packages seems like a bit of a heavyweight thing to do.

Reply:
Also answering your comment [here](https://github.com/ros-planning/moveit/pull/1051#discussion_r216137735) .
We are not planning to make gazebo a hard dependency in anyway, it is an optional simulator which is disabled by default. if we ended up modifying the current `demo.launch` , gazebo will only run if the user want that by passing an argument to the launch file.

Reply:
I suggest in the package.xml we add the following:

    <!-- Optional: Gazebo integration. This is a heavy dependency so is disabled by default -->
    <!-- run_depend>gazebo_ros</run_depend -->

Reply:
I agree, added.